### PR TITLE
Donator Item

### PR DESCRIPTION
**Pull request by jjpark-kb**
## About The Pull Request
adds Lil' Zapp, which is a donator plushie.

## Changelog

:cl:
add: added Lil' Zapp the plushie
/:cl:

### CODE REVIEW

**Comment by desertmote** [comment]
finally some pwr game merch

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -304,3 +304,71 @@
...
+/obj/item/toy/plush/zapp/attack_self(mob/user)
+	. = ..()
+	var/turf/src_turf = get_turf(src)
+	playsound(src_turf, 'sound/items/drink.ogg', 50, TRUE)
+	var/skill_level = user.mind.get_skill_level(/datum/skill/gaming)
+	if(user.ckey == "cameronlancaster" && skill_level <= 6)
+		say(skill_response[6])
+	else
+		say(skill_response[skill_level])
+	if(skill_level == 7)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/game/objects/items/plushes.dm]: I'm not sure what that logic is supposed to be, but I feel like there's something wrong going on here.

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm [hunk 2]
```diff
@@ -304,3 +304,71 @@
...
+	var/static/list/skill_response = list(
+		"Weak! What are you, a mobile gamer?",
+		"Come on, you can do better than that! Play some Orion Trial and try again.",
+		"Hey, not bad! Try and work on your APM.",
+		"Nice! You should see about competing in some local tournaments, gamer!",
+		"Now that's real skill! I think you deserve some Pwr Game.",
+		"Gamer God in the house! Look upon them and weep, console peasants!",
+		"Whoa! Gamer overload! Stand clear!!",
+	)
+	var/static/list/hit_response = list(
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/game/objects/items/plushes.dm]: I know that this is a plushie, but please document those new vars anyway :)

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm (hunk 1 above)
Comment (jjpark-kb) [reply modular_skyrat/modules/customization/game/objects/items/plushes.dm]: nothing is wrong here.

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm (hunk 2 above)
Comment (jjpark-kb) [reply modular_skyrat/modules/customization/game/objects/items/plushes.dm]: sure.

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm [hunk 3]
```diff
@@ -304,3 +304,74 @@
...
+		"This jacket isn't armored, you know!",
+		"I'm a collectible! You can't treat me like this!",
+		"Cut it out, or I'm telling chat!",
+	)
+
+/obj/item/toy/plush/zapp/attackby(obj/item/I, mob/living/user, params)
+	. = ..()
+	say(pick(hit_response))
+
+/obj/item/toy/plush/zapp/attack(mob/living/M, mob/living/user, params)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/game/objects/items/plushes.dm]:
```suggestion
/obj/item/toy/plush/zapp/attackby(obj/item/weapon, mob/living/user, params)
	. = ..()
	say(pick(hit_response))

/obj/item/toy/plush/zapp/attack(mob/living/attacked_mob, mob/living/user, params)
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Review by GoldenAlpharex — APPROVED** [review]

**Comment by jjpark-kb** [comment]
Shoot, I didn’t commit the changes…

**Review by GoldenAlpharex — APPROVED** [review]

**Comment by Elli-Skala** [comment]
Corporate marketing at it again

**Pull request merged**